packages/libdash/libdash.0.3/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Bindings to the dash shell's parser"
+maintainer: "Michael Greenberg <michael.greenberg@pomona.edu>"
+authors: "Michael Greenberg <michael.greenberg@pomona.edu>"
+license: "BSD-3-Clause"
+homepage: "https://github.com/mgree/libdash"
+bug-reports: "https://github.com/mgree/libdash/issues"
+depends: [
+  "ocaml" {>= "4.0.7"}
```

Comment:
Are you sure you mean 4.0.7 ? There is no such version of the compiler. Maybe you meant 4.07
```suggestion
  "ocaml" {>= "4.07"}
```